plugin/Installer/DatabaseChanges.php

```diff
@@ -138,6 +138,11 @@ public function install()
 			$dbversion = 16;
 		}
 
+		if ($dbversion == 16) {
+			dbDelta( $this->migrationsDataSimilarEstates() );
```

Comment:
The `dbDelta` WP function is not helpful here, because it takes a `CREATE TABLE` statement and compares it to the currently existing schema and modifies it into an `ALTER TABLE`. What's being done here is to just insert a row.
The `dbDelta` function call can just be removed and `$this->migrationsDataSimilarEstates()` should be called directly.